imports/plugins/core/ui/client/components/table/sortableTable.js

```diff
@@ -123,7 +126,7 @@ class SortableTable extends Component {
 
     // Add minWidth = undefined to override 100px default set by ReactTable
     const displayColumns = columnMetadata.map((element) => {
-      return _.assignIn({}, element, {
+      return _.extend({}, element, {
```

Comment:
We should use native ES2015 in place of lodash when we can. `Object.assign` should work in this case.